lib/v2/screens/sign_up/add_phone_number/add_phone_number.dart

```diff
@@ -34,7 +36,22 @@ class _AddPhoneNumberState extends State<AddPhoneNumber> {
       child: Scaffold(
         appBar: AppBar(),
         body: BlocConsumer<SignupBloc, SignupState>(
-          listener: (context, state) {},
+          listener: (context, state) {
+            if (state.addPhoneNumberState.pageState == PageState.failure) {
+              ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+                content: Text(
+                  state.addPhoneNumberState.errorMessage ?? 'Oops, something went wrong. Please try again later.',
+                  style: Theme.of(context).textTheme.subtitle2,
+                ),
+                backgroundColor: AppColors.red1,
+                padding: const EdgeInsets.symmetric(vertical: 8, horizontal: 16),
+              ));
```

Comment:
Do we have an SnackError? if not you must ask to Jason for color and styles. (also you can take SnackInfo as a reference). This could fit as a global component.